Directory.Packages.props

```
@@ -104,7 +104,7 @@
     <PackageVersion Include="MongoDB.Driver.Core.Extensions.DiagnosticSources" Version="1.4.0" />
     <PackageVersion Include="MySqlConnector.DependencyInjection" Version="2.3.5" />
     <PackageVersion Include="MySqlConnector.Logging.Microsoft.Extensions.Logging" Version="2.1.0" />
-    <PackageVersion Include="NATS.Net" Version="2.1.3" />
+    <PackageVersion Include="NATS.Net" Version="2.2.0" />
```

Looks like a new version is already available.

```suggestion
    <PackageVersion Include="NATS.Net" Version="2.2.1" />
```

---

Sure, I can bump it, but let's make sure it also gets bumped in main then, as main currently has 2.2.0 and this change won't get merged back into main.

---

> let's make sure it also gets bumped in main then

Done in https://github.com/dotnet/aspire/pull/3989.